development-defaults.ini

```diff
@@ -209,7 +209,7 @@ charity    = "Charity"
 chipspace  = "Chipspace"
 console    = "Consoles"
 panels     = "Panels"
```

Comment:
Unrelated change which I was asked ot make in Slack at the same time I was doing this.